test/integration/default/default_spec.rb

```diff
@@ -0,0 +1,406 @@
+# encoding: utf-8
+
+# Inspec test for CIS_2012r2_L1
+#
+# Copyright (c) 2017 Matt Tunny, All Rights Reserved.
+#
+# The Inspec reference, with examples and extensive documentation, can be
+# found at http://inspec.io/docs/reference/resources/
+
+# WinLogon Tests
+describe registry_key('HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Winlogon') do
```

Comment:
yeah, i'll start a pull request on breaking apart the tests in line with new ones added in here